<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignalSlotForm</class>
 <widget class="QWidget" name="SignalSlotForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Signal and Slot</string>
  </property>
  <widget class="QListWidget" name="lwSignalList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="lwSlotList">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>231</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
